docs/api/scheduler.md

```diff
@@ -1361,6 +1361,55 @@ Node details include host and rack name, capacity, resources, utilization, and a
 }
 ```
 
+## Nodes utilization
+
+Shows how nodes are distributed with regarding the utilization
+
+**URL** : `/ws/v1/scheduler/node-utilization`
+
+**Method** : `GET`
+
+**Auth required** : NO
+
+**Code** : `200 OK`
+
+**Content examples**
+
+```text
+{
+    "type": "vcore",
+    "utilization": [
+      {
+        "bucketName": "0-10%",
+        "numOfNodes": 1,
+        "nodeNames": [
+          "aethergpu"
+        ]
+      },
+      {
+        "bucketName": "10-20%",
+        "numOfNodes": 3,
+        "nodeNames": null
```

Comment:
If we show 3 nodes we should have a list of 3 nodes in the nodeNames also for consistency.

Reply:
Hi @wilfred-s 

Thanks for your review, I modified the `numOfNodes` to 2, and some example nodes data.